app/lib/lnd/util.js

```diff
@@ -163,15 +164,17 @@ export const waitForFile = (filepath, timeout = 1000) => {
 
   // Promise B resolves when the file has been found.
   let promiseB = new Promise(resolve => {
-    let intervalId = setInterval(() => {
+    let intervalId = setInterval(async () => {
```

Comment:
Not obvious why searching for file is wrapped into setInterval. Maybe add some comments?

Reply:
Have added some additional comments to this method